modules/nw-anp-audit-logging-concept.adoc

```diff
@@ -0,0 +1,136 @@
+// Module included in the following assemblies:
+//
+// * networking/openshift_network_security/AdminNetworkPolicy/logging-anp-policy.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="nw-anp-audit-logging_{context}"]
+= AdminNetworkPolicy audit logging
+
+Audit logging is enabled per `AdminNetworkPolicy` CR by annotating an ANP policy with the `k8s.ovn.org/acl-logging` key such as in the following example:
+
+.Example of annotation for `AdminNetworkPolicy` CR
+[%collapsible]
+====
+[source,yaml]
+----
+apiVersion: policy.networking.k8s.io/v1alpha1
+kind: AdminNetworkPolicy
+metadata:
+  annotations:
+    k8s.ovn.org/acl-logging: '{ "deny": "alert", "allow": "alert", "pass" : "warning" }'
+  name: anp-tenant-log
+spec:
+  priority: 5
+  subject:
+    namespaces:
+      matchLabels:
+        tenant: red
+  ingress:
+    - name: "allow-all-ingress-tenant-1"
+      action: "Allow"
+      from:
+      - pods:
+        namespaces:
+          namespaceSelector:
+            matchLabels:
+              tenant: blue
+        podSelector:
+          matchLabels:
+            custom-anp: tenant-1
+  egress:
+    - name: "deny-ns-tenant-1"
+      action: "Deny"
+      to:
+      - namespaces:
+           matchLabels:
+             tenant: blue
+    - name: "allow-ns-tenant-2"
+      action: "Allow"
+      to:
+      - namespaces:
+           matchLabels:
+             tenant: orange
+    - name: "pass-ns-tenant-3"
+      action: "Pass"
+      to:
+      - namespaces:
+           matchLabels:
+             tenant: pink
+----
+====
+
+Logs are generated whenever a specific OVN ACL triggers an event that meets the action criteria set in your logging annotation. In the example, an event in which any of the namespaces with the label `tenant: red` are denied egress to any of the namespaces with the label `tenant: orange`, a log is generated.
```

Comment:
```suggestion
Logs are generated whenever a specific OVN ACL triggers an event that meets the action criteria set in your logging annotation. In the example, an event in which any of the namespaces with the label `tenant: red` are denied egress to any of the namespaces with the label `tenant: orange`, a log is generated.
```
The 2nd sentence is confusing to me. If you are referring to the code sample above, then it seems incorrect because the orange tenant has an Allow action:

```
name: "allow-ns-tenant-2"
      action: "Allow"
      to:
      - namespaces:
           matchLabels:
             tenant: orange
```
if I'm totally misunderstanding, then I'd consider rephrasing/rewording the sentence for clarity.

Reply:
Good catch thank you.